mqtt/mqtt-bridge/src/persist/waking_state/memory/test.rs

```diff
@@ -0,0 +1,35 @@
+use std::{collections::VecDeque, num::NonZeroUsize, task::Waker};
+
+use mqtt3::proto::Publication;
+
+use crate::persist::{
+    waking_state::memory::WakingMemoryStore, Key, PersistResult, StreamWakeableState,
+};
+
+pub struct TestWakingMemoryStore(WakingMemoryStore);
```

Comment:
why do we need this wrapper?

Reply:
It just made tests easier for test_case as I could just type TestWakingMemoryStore::default() :)